server/src/unifyfs_global.h

```diff
@@ -197,5 +198,13 @@ unifyfs_rc disconnect_app_client(app_client* clnt);
 
 unifyfs_rc cleanup_app_client(app_client* clnt);
 
+/**
```

Comment:
Since `unifyfs_publish_server_pids()` is only used in `server/src/unifyfs_server.c`, I suggest you move this declaration to that file.

Reply:
I initially did that but guess checkpatch script was not happy. I will check it again.